lib/routes/account.js

```diff
@@ -525,7 +527,7 @@ module.exports = function (
             return db.consumeUnblockCode(emailRecord.uid, unblockCode)
               .then(
                 (code) => {
-                  if (Date.now() - code.createdAt > unblockCodeLifetime) {
+                  if (requestNow - code.createdAt > unblockCodeLifetime) {
```

Comment:
I believe we can continue to not poll the system for an updated time, and I'll just upgrade the test instead. It's a case of the test having its `Date.now()` called **after** the `Date.now()` in the start of the request, so the test's version is indeed younger.

Fix will be to just make the test no longer directly on the millisecond, but just test later in general.